src/api/python/speechd_config/config.py

```diff
@@ -667,8 +675,10 @@ def configure_basic_settings(self, type='user'):
 This is usually not necessary, most applications will start Speech Dispatcher automatically."""),
                 False)
             if setup_autostart:
-                os.system("""cp %s ~/.config/autostart/""" % os.path.join(buildconfig.SPD_DESKTOP_CONF_PATH,
-                                                                          "speechd.desktop"))
+                shutil.copy(
+                    os.path.join(buildconfig.SPD_DESKTOP_CONF_PATH, "speechd.desktop"),
+                    os.path.join(BaseDirectory.xdg_config_home, "autostart")
```

Comment:
The ending `/` is important to make sure that we put the file into the `autostart` directory, and not actually create an `autostart` file just because there wasn't any such entry before. So I guess just add `, "/"`